src/lr_apply.rs

```diff
@@ -154,7 +154,7 @@ fn lr_sbrow<BD: BitDepth>(
     let chroma = (plane != 0) as c_int;
     let ss_ver = chroma & (f.sr_cur.p.p.layout == Rav1dPixelLayout::I420) as c_int;
     let ss_hor = chroma & (f.sr_cur.p.p.layout != Rav1dPixelLayout::I444) as c_int;
-    let frame_hdr = &***f.frame_hdr.as_ref().unwrap();
+    let frame_hdr = &f.frame_hdr;
```

Comment:
... and an obviousish second step to improve this patch would be to clean up this kinda thing. emphasis on "do the broad strokes seem useful" :)